### ISSUE TITLE
Remove Page Borders

### ISSUE THREAD

**Issue by nlundby** [viktorhajer/simple-pdf-viewer#5]
It would be convenient to disable page borders on default because I would expect this module to be used more for mobile development and layouts need to be precise.

I haven't put too much work into it, but you can disable it by setting removePageBorders: false in 
\simple-pdf-viewer\esm2015\simple-pdf-viewer.js
\simple-pdf-viewer\esm5\simple-pdf-viewer.js

For full explanation of the issue see below:
https://github.com/mozilla/pdf.js/pull/5844/

**Comment by viktorhajer** [comment]
Okay, I will check it today.

**Comment by androidovshchik** [comment]
@nlundby 
Simply do with css
```
/deep/ .pdfViewer .page {
    margin: 0;
    border: none;
}
```

**Comment by viktorhajer** [comment]
True story, but I gonna create an input for this parameter (false by default) to make it easier.

**Comment by viktorhajer** [comment]
The small change was merged: 
https://github.com/viktorhajer/simple-pdf-viewer/pull/6/files

I updated the wiki pages and the docs. 

It will release later today.

**Comment by viktorhajer** [comment]
I've released the 2.0.0 version with the new input parameter. I planned to create a new demo page and print function but it will be postponed (but coming soon).
